app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java

```diff
@@ -285,6 +289,16 @@ public void onChanged(Spannable aUrl) {
         }
     };
 
+    private Observer<Spannable> mIsContentFeedObserver = new Observer<Spannable>() {
+        @Override
+        public void onChanged(Spannable o) {
+            String current = url.getValue().toString();
```

Comment:
Where does `url` come from?

Reply:
Okay, I think I see. The variable naming in this file is all over the place. We have a member variable name `url` but also local variables called `url`. This file really needs to be cleaned up.